tests/k6/src/test-issue-162.ts

```diff
@@ -13,11 +13,25 @@ export let options: Options = {
 };
 
 export default () => {
-    const res = api.uploadFile(defaults.accounts.einstein, files['kb_50.jpg'], `kb_50-${utils.randomString()}.jpg`)
+    const fileName = `kb_50-${__VU}-${__ITER}.jpg`
+    const res = api.uploadFile(defaults.accounts.einstein, files['kb_50.jpg'], fileName)
 
     check(res, {
         'status is 201': () => res.status === 201,
     });
 
     sleep(1);
-};
\ No newline at end of file
+};
+
+export const teardown = (): void => {
+  console.log("teardown")
+  for (let vu=1; vu <= options.vus; vu++) {
+    for (let iter=0; iter < options.iterations/options.vus; iter++) {
+      const res = api.deleteFile(defaults.accounts.einstein, `kb_50-${vu}-${iter}.jpg`)
+      check(res, {
+          // status could be either 204(if file was created) of 404(if file was not uploaded)
+          'status is 204 or 404': () => [204, 404].includes(res.status),
```

Comment:
in my opinion we should check the 204 and 404 individually and not within one check. Why do we expect 404 at all?

Reply:
@fschade , when trying with large number of users, the upload requests fail, so some file are not created. 
For. eg when trying with 500 vus and 100 iterations I get
```
 ✗ status is 201
     ↳  78% — ✓ 785 / ✗ 215
```
Also I could not get the list of failed uploads here from default function, which would be easier.
checking each of them separately still gives error on one of the checks so cant do that either